### PR TITLE
Fixed #501 - Added a check if command exists and timeout in restart command

### LINKED ISSUE #501: Restart command fires too fast
- Description
Restart command fires too fast so no notification is returned to the client. 

- Repro
Click restart

### PR DESCRIPTION
**Pull request by NotExpectedYet**
Fixes #501

### CODE REVIEW

File: server_src/utils/system-paths.utils.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+const { lookpath } = require("lookpath");
 
 const systemRoot = "./";
 const logFolder = "logs";
 
 function getLogsPath() {
   return join(systemRoot, logFolder);
 }
+async function checkIfFunctionExistsInPath(systemCommand) {
+  return await lookpath(systemCommand);
```
Comment (davidzwa) [inline server_src/utils/system-paths.utils.js:11]: Isnt this a bit over the top? Calling lookpath directly is clearer for me

Comment (NotExpectedYet) [reply server_src/utils/system-paths.utils.js:11]: Possibly, I was just thinking of future but I gather this is the only place we're actually checking paths currently so I can call it. Will amend.

Comment (NotExpectedYet) [reply server_src/utils/system-paths.utils.js]: Replaced it.

**Comment by NotExpectedYet** [comment]
I'll sort the merge conflict just been cleaning and preparing some munch aha

**Comment by NotExpectedYet** [comment]
Success!

**Review by davidzwa — APPROVED** [review]
So proud! First jest work in the pocket.

**Pull request merged**